test/unit/pusher_test.rb

```diff
@@ -263,16 +263,16 @@ class PusherTest < ActiveSupport::TestCase
       @cutter.stubs(:rubygem).returns @rubygem
       create(:version, rubygem: @rubygem, number: '0.1.1', summary: 'old summary')
       @cutter.stubs(:version).returns @rubygem.versions[0]
-      @rubygem.stubs(:update_attributes_from_gem_specification!)
-      GemCachePurger.stubs(:call)
+      @rubygem.expects(:update_attributes_from_gem_specification!).with(@cutter.version, nil)
```

Comment:
Although, `expects` has the same effect as `stubs`, we shouldn't use expects here. At least for purposes of our tests, expects always means we are testing an assertion.